tests/prepdata/test_supergraph_construct.py

```diff
@@ -0,0 +1,74 @@
+import torch
+
+from kale.prepdata.supergraph_construct import SuperEdge, SuperGraph, SuperVertex, SuperVertexParaSetting
+
+# create three supervertices
+node_feat = torch.randn(4, 20)
+edge_index = torch.tensor([[0, 1, 2, 3], [1, 2, 3, 4]])
+edge_type = torch.tensor([0, 0, 1, 1])
+edge_weight = torch.randn(4)
+
+
+def test_supervertex():
```

Comment:
No need to test `supervertex` separately as it is already covered in `test_gripnet.py`